.github/workflows/ci.yml

```diff
@@ -17,5 +17,9 @@ jobs:
         with:
           node-version: ${{ steps.get-versions.outputs.node }}
       - run: yarn install --frozen-lockfile
-      - run: yarn build
+      - run: yarn build --scope @mux-elements/*
```

Comment:
there's a benefit to running build on the examples, though, maybe it should be solved as part of the parallel build PR.
The main benefit I see is that it has caught some type of issues that were unnoticed in the main project.